docs/src/main/sphinx/release/release-467.md

```diff
@@ -0,0 +1,40 @@
+# Release 467 (5 Dec 2024)
+
+## General
+
+* Change default protocol for internal communication to HTTP/1.1 to address
+  issues with HTTP/2. ({issue}`24281`)
+* Return compressed results to clients by default when using the spooling
+  protocol. ({issue}`24332`)
+* Add application identifier `azure.application-id`, `gcs.application-id`, or
+  `s3.application-id` to the storage when using the spooling protocol. ({issue}`24361`)
+* Add support for OpenTelemetry tracing to the HTTP, Kafka, and MySQL event
+  listener. ({issue}`24389`)
+* Fix receiving stop signal in the launcher required for graceful shutdown. ({issue}`24380`)
```

Comment:
Fix incorrect handling of SIGTERM signal, which prevented the server from shutting down. cc @nineinchnick, is that accurate?

Reply:
I think so